### ISSUE TITLE
[Call for contributions] Feature parity between TensorRT and PyTorch backend - Sampling

### ISSUE THREAD

**Issue by juney-nvidia** [NVIDIA/TensorRT-LLM#3700]
- [x] greedy search  
  - [ ] beam search
  - [x] top-k sampling
  - [x] top-p sampling
  - [x] stop words
  - [ ] penalty
  - [ ] bad words
  - [ ] return context logits
  - [ ] return generation logitx
  - [ ] return logprobs
  - [ ] num_return_sequences
  - [ ] top_p_min
  - [ ] no_repeat_ngram_size
  - [ ] min_p

**Comment by karljang** [comment]
I'm closing this issue as completed. Thanks!
